### ISSUE TITLE
Rendering issue in Edge browser

### ISSUE THREAD

**Issue by timbjames** [knockout/knockout#2048]
Hi, now I am not sure if this is an Edge browser bug (I expect it is), or a knockoutjs bug, but I thought I would raise it here so that if anyone else is experiencing this then it might help.

There seems to be a very weird rendering issue happening in Edge when you have the style "text-transform: uppercase"

I have been able to put together some code to demonstrate this issue.

jsbin: http://output.jsbin.com/zowekubiwi
gist: https://gist.github.com/anonymous/f2a8c1c7d63ee52fd848dee134d097c8

If you run the code in Chrome/Firefox/IE9/IE10 you will see the counter incrementing. If you run the code in Edge (latest 13) you will see it render the first counter, and then freeze. If you view the underlying DOM, you can see it updating, and if you actually select the text it does then update in the browser.

**Comment by luismijimenez** [comment]
Hi Timb, I have the same problem and didn't know what, so thanks a lot because this weird issue made me lost a lot of time.

**Comment by timbjames** [comment]
Yeah it took me some time to find out the cause of this. The solution was just to sacrifice not having some parts of the page in uppercase in Edge browser, which isn't too bad.

If you are in the situation where your client might be shouting for the need for uppercase text, then another work around which I had in place first was selecting the text on the screen after each update.

Here is some code which I was using:

`function` selectElement(element) {
    if (window.getSelection) {
        var sel = window.getSelection();
        sel.removeAllRanges();
        var range = document.createRange();
        range.selectNodeContents(element);
        sel.addRange(range);
    } else if (document.selection) {
        var textRange = document.body.createTextRange();
        textRange.moveToElementText(element);
        textRange.select();
    }
}
function clearSelection() {
    if (window.getSelection) {
        window.getSelection().removeAllRanges();
    } else if (document.selection) {
        document.selection.empty();
    }
}`

So after your knockout observables update, run `selectElement(...);` then once this has run, call `clearSelection()`.
